forge/test/mlir/llama/utils/utils.py

```diff
@@ -27,3 +27,25 @@ def load_model(model_path="openlm-research/open_llama_3b", **kwargs):
     tokenizer = LlamaTokenizer.from_pretrained(model_path)
 
     return framework_model, tokenizer
+
+
+def load_llama32(model_id: str = "meta-llama/Llama-3.2-1B", **kwargs):
+
+    # Default token
+    token = kwargs.get("token", "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")
+
+    # Load the config
+    config = LlamaForCausalLM.config_class.from_pretrained(model_id, token=token)
```

Comment:
Can we merge these two llama loading functions into single one? Are there some major structural diffs?

Reply:
Done